src/Microsoft.Identity.Web.Certificate/DefaultCredentialsLoader.CustomSignedAssertion.cs

```diff
@@ -37,11 +37,40 @@ public DefaultCredentialsLoader(IEnumerable<ICustomSignedAssertionProvider> cust
             CustomSignedAssertionCredentialSourceLoaders = sourceLoaderDict;
         }
 
+        /// <summary>
+        /// Constructor for DefaultCredentialsLoader with both custom signed assertion providers and custom credential source loaders.
+        /// </summary>
+        /// <param name="customSignedAssertionProviders">Set of custom signed assertion providers.</param>
+        /// <param name="logger">ILogger.</param>
+        /// <param name="credentialSourceLoaders">Additional credential source loaders. Can override built-in loaders.</param>
+        public DefaultCredentialsLoader(
+            IEnumerable<ICustomSignedAssertionProvider> customSignedAssertionProviders, 
+            ILogger<DefaultCredentialsLoader>? logger, 
```

Comment:
Please add the logger at the end, to be consistent with the derived class and with the other constructors.

Reply:
Thanks. Done